docs/npwd/dev/FrameworkIntegration.md

```diff
@@ -0,0 +1,31 @@
+---
+id: framework_integration
+title: Framework Integration with NPWD
+sidebar_label: Framework Integration
+---
+
+NPWD was designed to be a framework agnostic resource that could work with any arbitary framework. To enable this,
+we expose the `newPlayer` & the `unloadPlayer` export functions to pass the player context to NPWD.
```

Comment:
```suggestion
we expose the `newPlayer` & the `unloadPlayer` export functions to pass the player context to NPWD.
```
Need to mention the net event for hiding UI when taking a photo. https://github.com/project-error/npwd/pull/403#issuecomment-986163547 This comment has an example.